saleor/static/dashboard/scss/components/_buttons.scss

```diff
@@ -1,42 +1,23 @@
-.btn-flat {
-  color: $secondary-color;
-  &.btn:hover {
-    color: #fff;
-  }
-}
-
 .btn {
   &:not(.btn-floating) {
     box-shadow: none;
   }
-  &-danger {
-    color: $error-color;
+  &-flat {
+    @extend .btn;
+    color: $secondary-color;
     &:hover {
-      background: $error-color;
-      color: #fff;
+      background: transparentize(#cccccc, .6);
```

Comment:
Wait, you do set background color on hover. This should be `:focus` and `:active` below is correct.